diff --git a/src/math.py b/src/math.py
index 1111111..2222222 100644
--- a/src/math.py
+++ b/src/math.py
@@ -1,2 +1,5 @@
 def add(left, right):
     return left + right
+
+def subtract(left, right):
+    return left - right
diff --git a/tests/test_math.py b/tests/test_math.py
index 3333333..4444444 100644
--- a/tests/test_math.py
+++ b/tests/test_math.py
@@ -1,4 +1,7 @@
 from src.math import add
+from src.math import subtract
 # math tests
 def test_add():
     assert add(1, 2) == 3
+
+def test_subtract():
+    assert subtract(3, 1) == 2
